front/src/styles/scss/applications/operationalStudies/_simulationresults.scss

```diff
@@ -39,15 +39,22 @@
 
   .osrd-simulation-container {
     position: relative;
-    background-color: white;
     padding: 1rem;
     border-radius: 4px;
 
     &.speedspacechart-container {
-      background-color: rgb(247, 246, 238);
+      margin-bottom: 28px;
```

Comment:
The margin is not quite this one on mockup : 

![image](https://github.com/user-attachments/assets/2a8ebe3d-36af-48ad-8e3b-9f69285648e0)

We have this right now : 

![image](https://github.com/user-attachments/assets/ecff8543-9c69-4c57-b4ed-a79dbc7dd27b)

Reply:
Done :white_check_mark: